tests/client/test_machine.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import tinychain as tc
+import unittest
+from sklearn.datasets import load_breast_cancer
+from sklearn.model_selection import train_test_split
+
+from testutils import ClientTest
+
+Dense = tc.tensor.Dense
+
+ENDPOINT = "/transact/hypothetical"
+LEARNING_RATE = tc.F32(0.01)
+MAX_ITERATIONS = 10
+NUM_EXAMPLES = 20
+
+
+class Logistic_Regression():
```

Comment:
I think it would be simpler to make this a function called maybe `regress` or `multi_regress`. That way you don't have to worry about initialization, inheritance, etc.

Reply:
can you explain more ?  does name affect anything ?